### ISSUE TITLE
Improve color contrast between text and background

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1663]
Relevant Components:

- [ ] Document Information
- [ ] File Upload
- [ ] Settings

File upload inputs and notes in submission view.

Description
There is not a significant color contrast between the text and the background on which it appears. The visual presentation of text and images of text should have a contrast ratio of at least 4.5:1: [Success Criterion 1.4.3 Contrast (Minimum)|https://www.w3.org/TR/WCAG21/#contrast-minimum].

Current Code
```<p ng-if="!fileValidationError" class="ng-binding ng-scope">Choose file here or drag and drop to upload</p>```

Solution
Edit the applicable CSS file to darken the link text (#A6A6A6 to #696969, for example)

**Comment by jcreel** [comment]
We will not want to validate the choices under the "Look and Feel" dialog in the settings - but the default/stock CSS should be made accessible.
